pkg/util/provider/machinecontroller/machine_safety.go

```diff
@@ -94,6 +94,7 @@ func (c *controller) reconcileClusterMachineSafetyAPIServer(key string) error {
 					}
 					machine.Status.LastOperation = v1alpha1.LastOperation{
 						Description:    "Machine Health Timeout was reset due to APIServer being unreachable",
+						ErrorCode:      "",
```

Comment:
I think we should remove these `""` ErrorCode initialization from everywhere, will keep the code cleaner